package.json

```diff
@@ -11,7 +11,6 @@
     "webmaker"
   ],
   "dependencies": {
-    "langmap": "0.0.14",
```

Comment:
that was what it was before and we changed it to specific versions because of heroku caching deep dependencies (which tbh I don't want to fix because it seems really complicated).